pgjdbc/src/test/java/org/postgresql/test/jdbc4/Binary2StringTest.java

```diff
@@ -0,0 +1,55 @@
+package org.postgresql.test.jdbc4;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.postgresql.PGStatement;
+import org.postgresql.jdbc.PreferQueryMode;
+import org.postgresql.test.TestUtil;
+import org.postgresql.test.jdbc2.BaseTest4;
+
+import org.junit.After;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class Binary2StringTest extends BaseTest4 {
```

Comment:
Please add the test to the relevant test suite. Otherwise the test is not executed.